components/raftstore/src/store/worker/pd.rs

```diff
@@ -771,7 +781,7 @@ impl SlowScore {
 
             inspect_interval,
             ratio_thresh: OrderedFloat(0.1),
-            min_ttr: Duration::from_secs(5 * 60),
+            min_ttr: Duration::from_secs(STORE_HEARTBEAT_DELAY_LIMIT),
```

Comment:
I dont think min_ttr related to STORE_HEARTBEAT_DELAY_LIMIT, it's the min decreasing duration setting value